hack/verify-golint.sh

```diff
@@ -92,38 +89,21 @@ else
 			echo "$err"
 		done
 		echo
-		echo 'Please fix the above errors. You can test via "golint" and commit the result.'
+		echo 'Please review the above warnings. You can test via "golint" and commit the result.'
+		echo 'If the above warnings do not make sense, you can exempt this package from golint'
+		echo 'checking by adding it to  hack/.golint_failures (if your reviewer is okay with it).'
```

Comment:
nit: double space in front of "hack/.golint_failures"

Additionally, we previously offered commands to copy/paste and add the file to `.linted_packages` and sort the file. Should we do that here to add to the `.golint_failures` file?